.vscode/configurationCache.log

```diff
@@ -0,0 +1 @@
+{"buildTargets":[],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[],"compilerArgs":[]},"fileIndex":[]}}
```

Comment:
Is the `.vscode` directory added on purpose?

Reply:
Nope!